app/controllers/users_controller.rb

```diff
@@ -1,5 +1,29 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find params[:id]
```

Comment:
dùng find_by thay vì find

Reply:
find và find_by khác nhau ntn?

Reply:
find nếu không tìm thấy record trong db thì trả về exception, còn find_by trả về nil

Reply:
dùng find_by đi em

Reply:
if you can't control exeption, the system is cracked. I suggest 2 way:
1. find: you must control exception
2. find_by: you must control case that has @user == nil